src/redteam/strategies/index.ts

```diff
@@ -63,6 +64,15 @@ export const Strategies: Strategy[] = [
       return newTestCases;
     },
   },
+  {
+    id: 'best-of-n',
+    action: async (testCases, injectVar, config) => {
+      logger.debug('Adding Best-of-N to all test cases');
```

Comment:
nit for later - I know we support the concept of filtering strategies to specific plugins. I am not sure where that happens and if this message makes sense all the time.

Reply:
good point